### PR TITLE
feat: add exploit prediction scoring system

### PR DESCRIPTION
**Pull request by acuas**
Refs: CYS-4434

### CODE REVIEW

File: opencve/commands/imports/__init__.py [hunk 1]
```diff
@@ -33,13 +33,8 @@ def import_data(confirm):
             info("Bye.")
             return
 
-    # Import the CWE list
```
Comment (mimatache) [inline opencve/commands/imports/__init__.py]: Why did we decide to not import these?

Comment (acuas) [reply opencve/commands/imports/__init__.py]: my bad

**Comment by ncrocfer** [comment]
Nice PR 👍 I saw you created it in OpenCVE then close it, I think it's a mistake from you.

@acuas @mimatache Can we know the usage you do of OpenCVE at Cyscale?

**Comment by acuas** [comment]
> Nice PR 👍 I saw you created it in OpenCVE then close it, I think it's a mistake from you.
> 
> @acuas @mimatache Can we know the usage you do of OpenCVE at Cyscale?

Thank you for the feedback! You're right, we did initially create it in OpenCVE and then closed it. We were exploring and testing a few CVE tools, including OpenCVE, to see how they might integrate with EPSS. If there's interest in the PR, we'd be happy to open a pull request in the original repository. However, please note that due to the size of our team, we might not be able to maintain it over the long term. Hope that clarifies things!

**Comment by ncrocfer** [comment]
Ok I see. I'm working on a new version of OpenCVE and the support of EPSS will be integrated, no ETA yet but I prefer you don't create a PR if you're not going to maintain it later.
This 2nd version of OpenCVE will provide licenses for MSSP, don't hesitate to contact for further information.
Nicolas

**Review by AndreiStefanie — APPROVED** [review]

**Pull request merged**